frontend/packages/client/src/components/CommunityCreate/StepOne.js

```diff
@@ -212,7 +226,55 @@ export default function StepOne({
                   }}
                 >
                   <Upload className="has-text-white" />
-                  <input {...getInputProps()} />
+                  <input {...getLogoInputProps()} />
+                </div>
+              ) : null}
+            </div>
+          </div>
+        </div>
+        <div className="columns">
+          <div className="column is-12">
+            <div
+              className="is-flex is-flex-direction-column is-align-items-center is-justify-content-center cursor-pointer rounded-lg border-dashed-dark"
```

Comment:
same here:
className={`is-flex is-flex-direction-column is-align-items-center is-justify-content-center cursor-pointer rounded-lg ${!banner ? 'border-dashed-dark': ''}`}